src/App.vue

```diff
@@ -131,7 +131,7 @@
 <style>
 
   html {
-    font-size: 135%;
+    font-size: 125%;
```

Comment:
why not 100%?

Reply:
1rem on 100% is equal to 16px. 125% is 20px - It is alot easier to calculate.